turbinia/task_manager.py

```diff
@@ -50,6 +50,11 @@
 
 # Define metrics
 SERVER_TASKS = Gauge('server_tasks', 'Turbinia Server Total Tasks')
+SERVER_TASKS_COMPLETED = Gauge(
```

Comment:
Have a look at the naming conventions for metrics.
https://prometheus.io/docs/practices/naming/

I think following that standard it makes sensen to:
- start every metric with turbinia_
- add a unit to each metrics (total these cases)
So
- turbinia_server_tasks_total
- turbinia_server_tasks_completed_total
- turbinia_server_jobs_create/resolved_total
- turbinia_server_requests_total (maybe add _received_ as well...)

Reply:
Done, also, total server requests is the same as the received one